src/TheFox/FlickrCli/Command/DownloadCommand.php

```diff
@@ -506,6 +516,12 @@ private function downloadPhoto(SimpleXMLElement $photo, string $destinationPath,
 
             /** @var SimpleXMLElement $photo */
             $photo = $xmlPhoto->photo;
+            
+            // update the timstamps of the file
+            $update_date = (int) $photo->dates['lastupdate'];
+            $taken_date = strtotime((string) $photo->dates['taken']);
```

Comment:
Please use

```php
strval($photo->dates['taken'])
```

instead of

```php
(string) $photo->dates['taken']
```

here. The same goes for `(int)`. Use `intval()` instead.